frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js

```diff
@@ -49,7 +49,7 @@ export default function determineBucketsForFlight(
   abortLimit?: number,
 ): void {
   const queryMatrix = M4x4.scale1(1, matrix);
-  const width = constants.VIEWPORT_WIDTH;
+  const width = 2 * constants.VIEWPORT_WIDTH;
```

Comment:
This is a workaround at the moment. Strictly speaking, we need the actual viewport size here, so that we know that sampling causes only artifacts below 10x10 pixel patches.

Reply:
This is a very good point. I thought it would be equal, but I just double checked, and it seems as if the bucket picker just sampled additional texels outside of the actual viewport with this change. In turn this means, that the change did only worsen the performance, but didn't change any actual rendering results? I removed the change now (and also kept iterStep at the old value). Let's wait for the lab testing to see whether any artifacts appear.

Reply:
Nice, even better, let's wait and see :)